core-bundle/src/Routing/Content/StringResolver.php

```diff
@@ -36,6 +36,10 @@ public function resolve(object $content): ContentUrlResult|null
 
         $url = $this->insertTagParser->replaceInline($content->value);
 
+        if ($url === '') {
+            return null;
```

Comment:
```suggestion
            throw new ForwardPageNotFoundException()
```

(also needs an import)